code/modules/mob/mob_defines.dm

```diff
@@ -153,11 +153,12 @@
 	var/list/shouldnt_see = list()	//list of objects that this mob shouldn't see in the stat panel. this silliness is needed because of AI alt+click and cult blood runes
 
 	var/mob_size = MOB_MEDIUM
+	var/throw_multiplier = 1 
```

Comment:
Miscommunication; this really really doesn't call for adding a new var at **/mob/ level** - it would affect *observers and AI eyes* which can't even be thrown, 99% of the time nobody will bother changing that, and it's kind of magical.